src/App.js

```diff
@@ -78,17 +80,45 @@ const App = () => {
 
   const resetGame = () => {
     // Complete in Wave 4
+    setSquares(generateSquares());
+    setCurrentPlayer(player1);
+    setWinningPlayer('...');
   };
```

Comment:
Yes, resetting the game essentially comes down to setting the state back to its initial values.